purchase_isolated_rfq/models/purchase_order.py

```diff
@@ -33,7 +33,6 @@ class PurchaseOrder(models.Model):
         ],
         string="RFQ Status",
         readonly=True,
-        related="state",
```

Comment:
@SurachetS Why remove this?
Note: I checked form v13, this is intended to be related field with store=False.

Note, we have new field rfq_state, but no value in it. And so, when I create a new RFQ, the state is not displayed.

![image](https://user-images.githubusercontent.com/1973598/105006783-afa7ae80-5a69-11eb-82b8-33b4cbcba04f.png)